src/corrplot.jl

```diff
@@ -11,7 +19,7 @@ function update_ticks_guides(d::KW, labs, i, j, n)
 end
 
 @recipe function f(cp::CorrPlot)
-    mat = cp.args[1]
+    mat = to_corrplot_matrix(cp.args[1])
```

Comment:
Should we just mimic Makie and have a `convert_arguments(::Type{CorrPlot}, args...)` as a method to overload? I find it odd to use a function specific to `CorrPlot`.

Reply:
Agreed, that would be better - would probably require a substantial modification to the recipe system, though.